roles/core/conman/tasks/main.yml

```diff
@@ -30,23 +30,66 @@
   tags:
     - package
 
+- name: group █ Ensure group conman exists
+  group:
+    name: conman
+    state: present
+    system: yes
+
+- name: user █ Ensure user conman exists
+  user:
+    name: conman
+    state: present
+    comment: "Connection Manager service"
+    group: conman
+    home: /var/run/conman
+    shell: /bin/false
+    system: yes
+
 - name: file █ Create /var/log/conman directory
   file: # noqa 208
     path: /var/log/conman
-    setype: conman_log_t
+    setype: "{{ (ansible_facts.selinux.status == 'enabled') | ternary('conman_log_t', omit) }}"
     state: directory
+    owner: conman
+    group: conman
     recurse: yes
+  notify: service █ Restart conman
```

Comment:
Is it relevant to restart conman here ?
This folder should be set at first deployment and is not expected to be removed later. Also, restarting now means conman will mostly try to start first time without the conman.conf generated because template is run later in the task.

Reply:
Yes this is relevant. First deployment was months ago for some users of the stack.

Reply:
> Also, restarting now means conman will mostly try to start first time without the conman.conf generated because template is run later in the task.

It does not restart "now". It notifies the handler. The restart will occur later, when the play will call the task `meta: flush_handlers`